tests/test_datasetattributes.py

```diff
@@ -11,6 +11,13 @@
 from pyvista.utilities import FieldAssociation
 
 
+@fixture()
+def sphere_empty_attributes(sphere):
+    ds_attr = hexbeam.point_arrays
```

Comment:
Sphere or hexbeam?

Reply:
This fixture isn't used and I forgot to remove it.